docs/source/troubleshooting/ssh.rst

```diff
@@ -85,7 +85,48 @@ runner boxes, and add the following configuration lines in ``/etc/st2/st2.conf``
 
   [ssh_runner]
   use_ssh_config = True
-  ssh_config_file_path = /home/stanley/.ssh/config
+  ssh_config_file_path = /root/.ssh/config
+
+Make sure your ssh config is in the same account as user running the st2action process.  If root is running 
+st2actions install it there.  Make sure the config and identity files have proper permissions and ownership.
+ 
+.. code-block:: bash
+
+  chown -R root:root /root/.ssh/*
+  chmod 600 /root/.ssh/config
+  chmod 600 /root/.ssh/id_rsa
+
+This is a sample ssh config that is known to work with bastion forwarding.
+
+.. code-block:: ini
```

Comment:
GitHub's [Linguist project](https://github.com/github/linguist) [supports syntax highlighting for SSH (client) configs](https://github.com/github/linguist/issues/3450), but you have to mark it as `ssh-config`:

```suggestion
.. code-block:: ssh-config
```

![SSH (client) config coloring screenshot](https://user-images.githubusercontent.com/597113/90189951-f682d100-dd72-11ea-87a3-225d9729d7c4.png)

And while you are at it, could you also fix this for line 160?

Leave line 84 alone though, since `/etc/st2/st2.conf` is just an INI file and is correctly marked as such. :+1:

Reply:
I just pushed a fix for this.

Reply:
Ah. We can't use `ssh-config` because this Sphinx uses pygments, not GitHub's linguist. So `ini` is the way to go.